src/main/scala/com/fulcrumgenomics/vcf/filtration/FilterSomaticVcf.scala

```diff
@@ -103,7 +128,12 @@ class FilterSomaticVcf
         else invalid("Must supply --sample when VCF contains more than one sample.")
     }
 
-    val filters = Seq(new EndRepairArtifactLikelihoodFilter(endRepairDistance, endRepairPValue))
+    if (aTailingDistance.isEmpty && endRepairDistance.isEmpty) logger.warning("All filters are turned off and tool will no-op!")
```

Comment:
I confirm that the tool will No-op because it is now possible to selectively turn on/off each of the two filters so a user could presumably turn them all to "off":

```bash
❯ java -jar fgbio-1.4.0-fe54cf7-SNAPSHOT.jar FilterSomaticVcf \
    --input input.vcf.gz \
    --output output.vcf.gz \
    --bam input.bam \
    --a-tailing-distance :none: \
    --end-repair-fill-in-distance :none:
```

```bash
❯ gzip -dc input.vcf.gz | grep -v '#' | wc -l
22260
```

```diff
❯ diff <( gzip -dc input.vcf.gz ) <( gzip -dc output.vcf.gz )
<empty>
```

I'll amend the documentation to make this functionality clear!

Reply:
resolved